dll/win32/browseui/CAutoComplete.cpp

```diff
@@ -279,6 +280,48 @@ HRESULT WINAPI CAutoComplete::SetOptions(DWORD dwFlag)
     return hr;
 }
 
+/* Edit_BackWord --- Delete previous word in text box */
+static void Edit_BackWord(HWND hwndEdit)
+{
+    INT iStart, iEnd;
+    iStart = iEnd = 0;
+    SendMessageW(hwndEdit, EM_GETSEL, (WPARAM)&iStart, (LPARAM)&iEnd);
+
+    if (iStart != iEnd)
+        return;
+
+    DWORD cchText = GetWindowTextLengthW(hwndEdit);
+    size_t cb = (cchText + 1) * sizeof(WCHAR);
+    LPWSTR pszText = (LPWSTR)CoTaskMemAlloc(cb);
```

Comment:
`CComHeapPtr` would auto-cleanup, removing the need for manual `CoTaskMemFree` in multiple paths.

Reply:
Fixed. Thanks.